examples/large_deformation/hyperelastic_tl_up_interactive.py

```diff
@@ -0,0 +1,394 @@
+r"""
```

Comment:
Could you add
```
#!/usr/bin/env python
```
to the first line, and make the script executable (`chmod a+x`), so that it can be executed on Linux without explicitly typing 'python'? All the other interactive examples have that.